fix: update podfile and gemfile cocoapods version

AkaShark opened the pull request:
pod install error

tisfeng commented:
> I have another problem
> 
> ```
> Resolve Package Graph
> package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0fatalError
> 2025-09-23 21:35:28.312 xcodebuild[14844:593740] Writing error result bundle to /var/folders/mx/h3wfrh5s5ms3dfzsh13hv3xr0000gn/T/ResultBundle_2025-23-09_21-35-0028.xcresult
> Resolve Package Graph
> package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0fatalError
> xcodebuild: error: Could not resolve package dependencies:
>   package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0
>   package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0
>   fatalError
> ```
> 
> Has our project been updated to Swift 6.1? I encountered this problem when updating SPM.

No, I briefly used [swift-subprocess](https://github.com/swiftlang/swift-subprocess) in SelectedTextKit before, as shown in this commit: [https://github.com/tisfeng/SelectedTextKit/commit/3624721da6ac24ce037a9c80c525568122702347]. 

But later I found that it caused [exceptions](https://github.com/tisfeng/Easydict/issues/981#issuecomment-3341505403) in release builds (it worked fine in debug builds), so I removed it and replaced it with NSAppleScript that swift-subprocess has been removed, it shouldn't cause any errors.

AkaShark commented:
> > I have another problem
> > ```
> > Resolve Package Graph
> > package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0fatalError
> > 2025-09-23 21:35:28.312 xcodebuild[14844:593740] Writing error result bundle to /var/folders/mx/h3wfrh5s5ms3dfzsh13hv3xr0000gn/T/ResultBundle_2025-23-09_21-35-0028.xcresult
> > Resolve Package Graph
> > package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0fatalError
> > xcodebuild: error: Could not resolve package dependencies:
> >   package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0
> >   package 'swift-subprocess' is using Swift tools version 6.1.0 but the installed version is 6.0.0
> >   fatalError
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Has our project been updated to Swift 6.1? I encountered this problem when updating SPM.
> 
> No, I briefly used [swift-subprocess](https://github.com/swiftlang/swift-subprocess) in SelectedTextKit before, as shown in this commit: [https://github.com/tisfeng/SelectedTextKit/commit/3624721da6ac24ce037a9c80c525568122702347].
> 
> But later I found that it caused [exceptions](https://github.com/tisfeng/Easydict/issues/981#issuecomment-3341505403) in release builds (it worked fine in debug builds), so I removed it and replaced it with NSAppleScript that swift-subprocess has been removed, it shouldn't cause any errors.

Yeah, you‘re right, it might be because my branch is backward or there’s  a Xcode caching problem. Thanks for your help.
